# Setup problem/game parameters before running the dlp_node
debug: False
# unique agent ID, starts from 0
myID: 0

# [rows, columns]
grid_size: [7,7]

# sector size (x,y) in ENU, in meter(s)
# for hawrdware test
#sector_size: [0.85, 0.85]
# for simulation
sector_size: [2, 2]

# origin shifts (dx,dy) from (0,0) in ENU, in meter(s)
# for hardware experiment
#origin_shifts: [3.0, 3.0]
# for simulation
origin_shifts: [0, 0]

N_defenders: 3
N_attackers: 1

# count of base sectors
nBase: 1

# set of base sectors 
Base: [1]

# count of base reference sectors
nBaseRef: 3

# set of base referense sectors
BaseRef: [2,8,9]

# strategy weights
alpha: -0.99
beta: -0.01

# prediction horizon
Tp: 2

neighbor_radius: 1


update_freq: 50

# home position
home: [1.0, 1.0]

# fixed altitude setpoint
altitude_setpoint: 1.0

# use joystick
enable_joystick: False

# whether to use GPS or directly local_position coordinates
# False for hardware test
use_gps: True

# whether to used simulated position, set True if using gazebo
# False for hardware test
use_sim: True

# lat/long of zero position
lat0: 47.397742
long0: 8.5455933

# fence for simulation
fence_min_x: 0
fence_max_x: 14.0
fence_min_y: 0.0
fence_max_y: 14.0

# for hardware test
#fence_min_x: -2.8
#fence_max_x: 2.8
#fence_min_y: -2.0
#fence_max_y: 2.2

# attacker capture distance, [meter]
# for hardware test
#capture_distance: 1.2
# for simulation
capture_distance: 2.1

# if true, use algorithm to compute attack plan
# otherwise, joystick is expected to be used
auto_attack: true
